### ISSUE TITLE
How do I speed up/slow down audio using CScore?

### ISSUE THREAD

**Issue by ADAMPOKE111** [filoe/cscore#198]
How do I speed up/slow down audio using CScore? Similar Audacity's Effect -> Change Speed, I was wondering if this is possible in CScore to automate things and make everything easier for me so I don't have to use Audacity.

**Comment by filoe** [comment]
I'm sorry. It is definitly a todo but not implemented yet.

**Comment by ArizonaInkStudios** [comment]
I wrote a custom EffectSource wrapping SoundTouch DLL to change tempo and pitch without changing the audio sound. With that EffectSource, I can change the tempo without it sounding like chipmunks and I can change the pitch without distortion. Might be a solution for you to try until CSCore supports it natively?

**Comment by filoe** [comment]
Sounds great. Maybe you could include a sample via pull request?

**Comment by ArizonaInkStudios** [comment]
Yes I could, but I was not sure about including SoundTouch as it is someone else's library. I will see what I can do though.

**Comment by ArizonaInkStudios** [comment]
I submitted a pull request. The files may not be where you would put them in your project (all developers organize differently) but you can see what I did. I also added a post build command to get the SoundTouch.dll copied over. Let me know if you have any questions.
